build_tooling/cleanup_storage.py

```diff
@@ -0,0 +1,16 @@
+from arcticdb import Arctic
+
+from storage_common import *
+
+if __name__ == "__main__":
+    # TODO: Add support for other storages
+    uri = get_real_s3_uri()
+    BRANCH_NAME = os.getenv("ARCTICDB_PERSISTENT_STORAGE_BRANCH_NAME")
```

Comment:
I wonder if there's any sort of UUID for a Github workflow run? Just slightly worried about races with concurrent builds of the same branch

Reply:
Shouldn't be a problem as with the current setup we cannot have 2 builds of the same branch running at the same time.